src/Mod/BIM/nativeifc/ifc_tools.py

```diff
@@ -622,9 +622,9 @@ def add_properties(
                 obj.addProperty("App::PropertyString", attr, "IFC")
             if value is not None:
                 setattr(obj, attr, str(value))
-    # link Label2 and Description
-    if "Description" in obj.PropertiesList and hasattr(obj, "setExpression"):
-        obj.setExpression("Label2", "Description")
+    # link Description  and Description
+    if "Description" not in obj.PropertiesList and hasattr(obj, "setExpression"):
+        obj.setExpression("Description ", "Description")
```

Comment:
I this intended? Looks like a circular reference.

Reply:
Should I just remove these lines altogether?

Reply:
625-627 lines has no sense at all. If Description exists, 627 will not execute due to the if condition. And if Description do not exists, you can't bind an expression to a non existent Property and worst than that you cannot bind an expression to itself. So i bet yes, remove them.

Reply:
Done

Reply:
So what happens with existing files that had the "Description" property and this expression set?